react/fields/MonthPicker/NativeMonthPicker/NativeMonthPicker.less

```diff
@@ -0,0 +1,59 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  position: relative;
+}
+
+.input {
+  .touchableText();
+  display: block;
+  position: relative;
+  width: 100%;
+  height: @grid-row-height * 5;
+  padding: 0 @field-gutter-width;
+  background: @sk-white;
+  border: @field-border-width solid @sk-mid-gray-light;
+  border-radius: @field-border-radius;
+  color: @sk-charcoal;
+  background-color: @sk-white;
+
+  &::-webkit-inner-spin-button,
+  &::-webkit-calendar-picker-indicator,
+  &::-webkit-clear-button {
+    display: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
```

Comment:
I'm checking on autoprefixer, I was under the impression appearance was more specific and wouldn't be auto-prefixed. However, I do need to move the `-moz-apearance` out of this selector.